requirements_test.txt

```diff
@@ -1,5 +1,6 @@
 -r requirements_test_min.txt
 -r requirements_test_pre_commit.txt
+contributors-txt>=0.7.3
```

Comment:
It wasn't there because it was not compatible with python 3.7, there's the runtime pip install to remove in the tbump configuration now that it is :)

Reply:
```suggestion
contributors-txt>=0.7.4
```

Reply:
Ah that's another reason not to wait too long with moving `pylint` to `3.7`: we can't do `astroid` until `pylint` has deprecated it.
Seeing the time it takes to merge a PR in `astroid` if we don't start the process soon we'll have `3.6` unsupported for `astroid` somewhere in 2025 😅